Recon/test/ecal_preselection_skim.py

```diff
@@ -0,0 +1,17 @@
+import os import sys
```

Comment:
Oops! looks like the `clang-format` was run on this Python file. Reverting this file to the commit before formatting is probably what you want to do.

It looks like the only change I'd suggest is to avoid using the pre-created skimmer (that I suggest to avoid creating) and instead create your own.

```python
from LDMX.Recon.ecalPreselectionSkimmer import EcalPreselectionSkimmer
eps = EcalPreselectionSkimmer("ecal-preselection")
eps.summed_tight_iso_mas = 1100.
eps.n_readout_hits_max = 90
p.sequence = [ eps ]
```

Reply:
oh god haha, ok fixing this now